lib/src/main/kotlin/com/swmansion/starknet/data/serializers/StringWrapperSerializer.kt

```diff
@@ -0,0 +1,21 @@
+package com.swmansion.starknet.data.serializers
+
+import com.swmansion.starknet.data.types.StringWrapper
+import kotlinx.serialization.KSerializer
+import kotlinx.serialization.descriptors.PrimitiveKind
+import kotlinx.serialization.descriptors.PrimitiveSerialDescriptor
+import kotlinx.serialization.descriptors.SerialDescriptor
+import kotlinx.serialization.encoding.Decoder
+import kotlinx.serialization.encoding.Encoder
+
+object StringWrapperSerializer : KSerializer<StringWrapper> {
```

Comment:
the same remark about the name, not sure StringWrapper is the best choice

Reply:
Wdyt of `HttpBatchRequestString`?

Reply:
as per comment with Int